cmd/tools/grafana/grafana_test.go

```diff
@@ -46,3 +47,45 @@ func TestHttpsAddr(t *testing.T) {
 		t.Errorf("Expected opts.addr to be %s but got %s", "https://1.1.1.1:3000", opts.addr)
 	}
 }
+
+// tests function addPrefixToMetricNames()
+func TestAddPrefixToMetricNames(t *testing.T) {
```

Comment:
YES! Thanks for adding tests for this 🎊

Reply:
I also wrote a test for `addGlobalPrefix()`, but seems like marshaling/unmarshaling JSON is a bit too complicated for reliable testing.